packages/cms-base-layer/components/SwProductReviews.vue

```diff
@@ -12,12 +16,14 @@ const props = defineProps<{
 type Translations = {
   product: {
     noReviews: string;
+    reviewNotAccepted: string;
   };
 };
 
 let translations: Translations = {
   product: {
     noReviews: "No reviews yet.",
+    reviewNotAccepted: "Your review has not been approved yet",
```

Comment:
Translations should also be added to the translation files

Reply:
this is moved from override component from template, not sure why it was created there, but cms-base should handle it. 
Translations are already there in files